client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -177,6 +197,9 @@ export function PatternPreview( props: PatternPreviewProps ) {
 			handleWrapperClass="pattern-preview__resizer"
 			minWidth={ 375 }
 			maxWidth="100%"
+			onResizeStop={ () => {
+				recordResizeEvent( 'calypso_pattern_library_viewport_resize' );
```

Comment:
The description mentions a different name, i.e. `calypso_pattern_library_resize`. I'm not sure of that `viewport` mention since the user is resizing a pattern rather than the width of their browser.

Reply:
Oh, good catch. The mention of `viewport` came from the event descriptions in p1711117310239469/1711087204.772069-slack-C06ELHR6L9J, and I didn't update it properly when copying testing instructions into this PR from another similar one.

@wongasy, can you confirm if we want `calypso_pattern_library_resize` or `calypso_pattern_library_viewport_resize` here? Stephane makes a good point that it isn't actually the viewport size we're tracking changes on.

Reply:
Sorry for the confusion. Let's go with `calypso_pattern_library_resize`, I've updated the requirement post according as well.